src/components/LoansByCategory/MFIRecommendations/MfiLoansWrapper.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+	<div class="tw-my-4 tw-px-2 lg:tw-px-0">
+		<h4 class="tw-text-h4 tw-text-action">
+			Support Borrowers from Fundación Pro Mujer
+		</h4>
+		<kiva-classic-loan-carousel-exp
+			:is-visible="true"
+			:loan-ids="selectedChannelLoanIds"
+			:selected-channel="selectedChannel"
+			:show-view-more-card="true"
+			:is-personalized="true"
+			:is-mfi="true"
+			id="carousel_exp"
+			class="-tw-mt-4"
+		/>
+
+		<div class="tw-flex tw-justify-center tw-mt-4">
+			<!-- eslint-disable max-len -->
+			<a class="tw-text-action tw-text-base" href="https://www.kiva.org/lend?partner=59&status=fundRaising&sortBy=newest">View All</a>
```

Comment:
Do we want this URL to be environment specific instead of just hitting prod for all environments?

Reply:
We could drop the domain `https://www.kiva.org` but that would make it harder to test in local dev ;) At least prod resolves!